tests/test_integration.py

```diff
@@ -532,3 +535,82 @@ def create_layout():
 
         self.startServer(app)
         time.sleep(0.5)
+
+    def test_with_custom_renderer(self):
+        app = dash.Dash(__name__)
+
+        app.index_string = '''
```

Comment:
Can we add another test that does the same thing but uses the `interpolate_index` method?

Reply:
Good point. I'm having a bit of trouble writing a test for this - could you take a look at the included test `test_with_custom_renderer_interpolated`? I'm not sure what's going on there, I'm getting JS errors about `window.push('load_after1')` not working for some reason.